lib/delayed/priority.rb

```diff
@@ -82,12 +82,39 @@ def alerts=(alerts)
         @alerts = alerts&.sort_by { |k, _| names.keys.index(k) }&.to_h
       end
 
+      def assign_at_midpoint?
+        @assign_at_midpoint || false
+      end
+
+      def assign_at_midpoint=(assign_at_midpoint)
+        @assign_at_midpoint = assign_at_midpoint
+      end
```

Comment:
I can take a shot at adding some documentation for this if this is the direction we end up going in